### DISCUSSION TITLE
Cache hit

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#12728] [Archive]
Hi!

Is there a way to check/determine if a response from directus is a cache hit?

I've set up redis as a caching mechanism, but unsure if it works. Any other way of checking if it works is also fine, but would be good to check in the response if that specific api call was a hit.

Regards,

Kees.

**Comment by rijkvanzanten** [comment]
There's currently no way to tell from the http response whether or not the value was served from the cache 🤔 The only way to tell from that side would be to look at the time of the request. The first request will take significantly longer than subsequent calls

**Reply by keesvanbemmel** [reply]
Hope this is to your liking: https://github.com/directus/directus/pull/12732

:)
